From 6e4b27ef13f2fe8c7179683b015fc8c09e7bfed5 Mon Sep 17 00:00:00 2001
From: Yang Shi <yang.shi@windriver.com>
Date: Thu, 5 Sep 2013 14:12:58 -0700
Subject: [PATCH] libhugetlbfs: Fix typo error in Makfile

CQID: WIND00433622

There is a typo error in Makefile for x86 build.

Signed-off-by: Yang Shi <yang.shi@windriver.com>
---
 Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index c29a523..2b308a3 100644
--- a/Makefile
+++ b/Makefile
@@ -64,7 +64,7 @@ else
 ifeq ($(ARCH),i586)
 CC32 = $(CC)
 ELF32 = elf_i386
-TMPLIB32 ib
+TMPLIB32 = lib
 else
 ifeq ($(ARCH),x86_64)
 CC64 = $(CC) -m64
-- 
1.7.5.4

